types/react-auth-kit/OTHER_FILES.txt

```diff
@@ -0,0 +1 @@
+TokenObject.d.ts
```

Comment:
You probably (?) mean to reference this in `index.d.ts` - you shouldn't need to list it here (and therefore shouldn't need this file?).